lib/puppetserver/ca/action/enable.rb

```diff
@@ -97,9 +97,10 @@ def create_infra_crl_chain(settings)
           return signer.errors if signer.errors.any?
 
           ca = LocalCertificateAuthority.new(signer.digest, settings)
-          infra_crl = ca.create_crl_for(ca.cert, ca.key)
           return ca.errors if ca.errors.any?
 
+          infra_crl = ca.create_crl_for(ca.cert, ca.key)
+
```

Comment:
Could add another call to `return ca.errors if ca.errors.any?` here, but as far as I can tell `create_crl_for` does not add anything to the `errors` array if a failure occurs.